nav2_rrtconnect_planner/include/nav2_rrtconnect_planner/rrt_connect.hpp

```diff
@@ -0,0 +1,130 @@
+/*********************************************************************
+ *
+ * Software License Agreement (BSD License)
+ *
+ *  Copyright (c) 2008, 2013, Willow Garage, Inc.
```

Comment:
Is this willow code?

Reply:
I thought about the ROS2 platform. I don't have much experience with licensing so please guide me.

Reply:
If you copy pasted code from somewhere, you should add their license. If you made this yourself, then you don't.